Fix "warning: using the result of an assignment as a condition without parentheses"
--- xbase64/xbmemo.cpp.orig	2006-07-17 11:54:42.000000000 -0500
+++ xbase64/xbmemo.cpp	2019-10-01 22:11:36.000000000 -0500
@@ -255,7 +255,7 @@
   if (ext == "DBF")
     MemofileName += "FPT";
   else
-    if (ext = "dbf")
+    if (ext == "dbf")
       MemofileName += "fpt";
     else
       return XB_INVALID_NAME;
